README.md

```diff
@@ -17,22 +17,25 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
 
 For more information, see [license](LICENSE.md).
 
-
 ## Deployment
 
-Refer to [docs.18f.gov](https://docs.18f.gov/getting-started/setup/) for getting set up with Cloud Foundry
-
+Refer to [docs.18f.gov](https://docs.18f.gov/getting-started/setup/) for getting
```

Comment:
my dotfiles are set up to automatically break at 80 chars - this was over that. Can undo if you like...